delfi_attributes.json

```diff
@@ -0,0 +1,1022 @@
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "elevator"
+		}
+	  },
+	  "TAGS": {
+		"level": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2100,
+	  "LABEL": "Stufe vorhanden",
+	  "CONDITION": [
+		{
+		  "osm_tags": {
+			"highway": "steps",
+			"step_count": "1"
+		  }
+		},
+		{
+		  "osm_tags": {
+			"barrier": "kerb",
+			"kerb": [
+			  "lowered",
+			  "flush",
+			  "raised",
+			  "no"
+			]
+		  }
+		}
+	  ],
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2101,
+	  "LABEL": "Stufenhöhe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "step_count": "1"
+		}
+	  },
+	  "TAGS": {
+		"step:height": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2101,
+	  "LABEL": "Stufenhöhe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "barrier": "kerb",
+		  "kerb": [
+			"lowered",
+			"raised"
+		  ]
+		}
+	  },
+	  "TAGS": {
+		"kerb:height": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2110,
+	  "LABEL": "Treppe vorhanden",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": false
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2111,
+	  "LABEL": "Verbindungsfunktion der Treppe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": false
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"level": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2112,
+	  "LABEL": "Stufenhöhe (Tritthöhe)",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": false
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"step:height": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2113,
+	  "LABEL": "Anzahl der Treppenstufen",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": false
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"step_count": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2120,
+	  "LABEL": "Rampe und Wege mit Neigungen vorhanden/Zugang über Rampe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": [
+			"footway",
+			"path"
+		  ],
+		  "incline": true
+		}
+	  },
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2121,
+	  "LABEL": "Verbindungsfunktion",
+	  "CONDITION": {
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"level": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2123,
+	  "LABEL": "Rampenbreite",
+	  "CONDITION": {
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"width": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2130,
+	  "LABEL": "Rolltreppe vorhanden",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": true
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {}
+	},
+	{
+	  "DELFI_ID": 2131,
+	  "LABEL": "Verbindungsfunktion der Rolltreppe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": true
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"level": []
+	  }
+	},
+	{
+	  "DELFI_ID": 2132,
+	  "LABEL": "Richtung der Rolltreppe",
+	  "CONDITION": {
+		"osm_tags": {
+		  "highway": "steps",
+		  "conveying": true
+		},
+		"osm_element": "OpenWay"
+	  },
+	  "TAGS": {
+		"incline": []
```

Comment:
I know that the wiki states this, but looking at the DELFI catalogue what do you think is correct here?

I feel like

- 2132 is `conveying=forward/backward`
- 2133 is `conveying=reversible`

Because an escalator can have `incline=up` or any positive value while still only being usable "downwards". In other words `conveying=yes` and `incline=*` doesn't tell me the direction the escalator is moving.

If you agree with me then we should also adjust the wiki.

Reply:
[Discussion] Valid values for 2132 and 2133 are forward/backward/reversible